### DISCUSSION TITLE
Mobile app wrongly reports that someone requested my review

### DISCUSSION THREAD

**Discussion by leonardoalt** [community/community#36716] [Projects and Issues]
Hi, not sure this is the right place for this but I couldn't find a better place so:

The GH Android app shows that someone requested my review in a PR:

![image](https://user-images.githubusercontent.com/504195/196822202-28d357ae-7fe0-4c99-b912-243c34805e91.png)

But if you open the PR (https://github.com/ethereum/solidity/pull/13636) you'll see the person actually requested reviews from 2 other people but not me. I've seen this consistently over multiple PRs.

**Comment by ossanna16** [comment]
Hey there @leonardoalt and thank you for asking a good question! I'm glad you found our community 🙂
